torch/_higher_order_ops/utils.py

```diff
@@ -129,7 +130,8 @@ def _has_potential_branch_input_mutation(branch, inputs, pre_dispatch=False):
     bit restrictive as the branch must be traceable.
     """
     try:
-        gm = make_fx(branch, pre_dispatch=pre_dispatch)(*inputs)
+        with disable_sym_dispatch():
```

Comment:
Slightly uncomfortable with `disable_sym_dispatch` being everywhere, but this is more of our fault that we use make_fx everywhere